x/amm/keeper/update_pool_for_swap.go

```diff
@@ -221,6 +221,11 @@ func (k Keeper) UpdatePoolForSwap(
 		)
 	}
 
+	if !pool.PoolParams.UseOracle {
+		tokenInPrice, tokenOutPrice := k.GetEstimatedTokensPriceFromBestPool(ctx, tokenIn.Denom, tokenOut.Denom)
+		types.EmitSwapPriceChangeEvent(ctx, pool.PoolId, tokenIn.Denom, tokenInPrice.String(), tokenOut.Denom, tokenOutPrice.String())
```

Comment:
the price should be of the pool in which swap is being done. This can report two prices - both different from the price at which swap was done